core/src/services/mod.rs

```diff
@@ -144,3 +144,8 @@ pub use onedrive::Onedrive;
 
 #[cfg(feature = "services-webhdfs")]
 pub use webhdfs::Webhdfs;
+
+// #[cfg(feature = "services-supabase")]
+mod vercel;
```

Comment:
Maybe we can use `vecel_artifacts` for short. `Vercel Remote Cache` is built upon `artifacts` related APIs.